zerver/lib/emoji.py

```diff
@@ -43,11 +43,20 @@ def translate_emoticons(text: str) -> str:
 
     return translated
 
-def emoji_name_to_emoji_code(realm: Realm, emoji_name: str) -> Tuple[str, str]:
-    realm_emojis = realm.get_active_emoji()
-    realm_emoji = realm_emojis.get(emoji_name)
+def emoji_name_to_emoji_code(realm: Realm, emoji_name: str,
+                             allow_deactivated: bool=False) -> Tuple[str, str]:
+    realm_emoji = None
+    if allow_deactivated:
+        realm_emojis = get_realm_emoji_dicts(realm)
+        for emoji in realm_emojis.values():
+            if emoji['name'] == emoji_name:
+                realm_emoji = emoji
```

Comment:
This is roughly O(n+n). (n for the get_realm_emoji_dicts and n for the loop). But the get_active_realm_emoji counterpart is O(n + logn) (The dictionary is created in such a way to facilitate logn find). I reckon we can do something similar for the deactivated part. @timabbott thoughts?